samples/matter/light_switch/src/chip_project_config.h

```diff
@@ -10,3 +10,11 @@
 /* Use a default pairing code if one hasn't been provisioned in flash. */
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 20202021
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
+
+/* When WiFi low power mode is enabled, 
+ * increase MRP local active retry interval to work with 
+ * 3 DTIMs and the Beacon interval equal to 102.4 ms.
+*/
+#ifdef NRF_WIFI_LOW_POWER
+    #define CHIP_CONFIG_ALLOW_RUNTIME_MRP_OVERRIDE 1
```

Comment:
It would be great to have some Kconfig layer that would set this value for our platform instead of doing it in the application, but maybe it's not a must have for now.

Reply:
I can add it to the Kconfig file in connectedhomeip SDK, but what is the advantage of having it there? To be consistent with other configs?